webapi/_test_data/ontologies/anything-onto.ttl

```diff
@@ -70,7 +70,9 @@
 
                salsah-gui:guiOrder "1"^^xsd:integer ;
 
-               salsah-gui:guiElement salsah-gui:Searchbox .
+               salsah-gui:guiElement salsah-gui:Searchbox ;
+	       
+	       salsah-gui:guiAttribute "size=45".
```

Comment:
it is optional, so there is no need to change any ontologies.
(plus, who knows how salsah v2 will make use of this)